packages/editor/src/components/collab-sidebar/style.scss

```
@@ -131,3 +131,80 @@
 		padding: 0;
 	}
 }
+
+// Comment avatar indicators
+.comment-avatar-indicator {
+	position: relative;
+	padding: 4px !important;
+	min-width: auto !important;
+	background: transparent !important;
+	border: none !important;
+
+	&:hover {
+		background: rgba(0, 0, 0, 0.04) !important;
+		border-radius: 4px;
+	}
+
+	&.has-unresolved {
+		.comment-avatar-stack {
+			&::after {
+				content: "";
+				position: absolute;
+				top: -2px;
+				right: -2px;
+				width: 8px;
+				height: 8px;
+				background: #d63638;
+				border-radius: 50%;
+				border: 1px solid $white;
```

```suggestion
				width: $grid-unit-10;
				height: $grid-unit-10;;
				background: #d63638;
				border-radius: $radius-round;
				border: $border-width solid $white;
```

Let's use [SCSS variables](https://github.com/WordPress/gutenberg/pull/71271/files#r2340638916) as much as possible. This also applies to the other styles in this file.

---

Will correct the other instances